### PR TITLE
Controlling the member card style using the use TextDirection hook

### PR DESCRIPTION
**Pull request by develad**

### CODE REVIEW

**Comment by develad** [comment]
Hi @Tamir198 
About #252  I know you opened a PR #253 but please check this implementation and tell me what you think?

File: app/[locale]/not-found.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function NotFound() {
       <p>{t('paragraph')}</p>
       <Link
         href={LINKS.HOME}
-        className="w-[250px] my-8 text-xs discord-btn rounded-full px-7 py-2 flex items-start justify-center gap-2 dark:hover:bg-purple-200 item-hover-transition items-center"
+        className="w-[250px] my-8 text-xs discord-btn rounded-full px-7 py-2 flex justify-center gap-2 dark:hover:bg-purple-200 item-hover-transition items-center"
```
Comment (develad) [inline app/[locale]/not-found.tsx:16]: Removing redundant class to clear problem from the the terminal

**Comment by develad** [comment]
This PR was merge into PR #250

**Comment by Tamir198** [comment]
@develad  

Generally, this would be the better solution, but for this case where the direction should be the same regardless of the language, in my opinion, it's a bit overkill.

**Comment by develad** [comment]
> @develad
> 
> Generally, this would be the better solution, but for this case where the direction should be the same regardless of the language, in my opinion, it's a bit overkill.

@Tamir198 

Why does the 'direction should be the same regardless of the language' ???
It doesn't specified in the Figma design. Only the Hebrew version of the card is  specified and it is only logical that the English version would be the other way around.

**Comment by Tamir198** [comment]
> > @develad
> > Generally, this would be the better solution, but for this case where the direction should be the same regardless of the language, in my opinion, it's a bit overkill.
> 
> @Tamir198
> 
> Why does the 'direction should be the same regardless of the language' ??? It doesn't specified in the Figma design. Only the Hebrew version of the card is specified and it is only logical that the English version would be the other way around.

I guess you can say it, I was thinking that it should be displayed like this because of the figma (which as you mentioned doesn't specify the English version).
Can you add a screenshot of how it looks?

**Pull request closed without merging**